src/tasks/harbor/harbor.ts

```diff
@@ -162,18 +163,70 @@ async function createTeamRobotAccount(projectName: string): Promise<RobotCreated
 
   if (existing?.id) {
     const existingId = existing.id
-    await doApiCall(errors, `Deleting previous robot account ${fullName}`, () => robotApi.deleteRobot(existingId))
+    await doApiCall(errors, `Deleting previous pull robot account ${fullName}`, () => robotApi.deleteRobot(existingId))
   }
 
-  const robotAccount = (await doApiCall(errors, `Creating robot account ${fullName} with project level perms`, () =>
-    robotApi.createRobot(projectRobot),
+  const robotPullAccount = (await doApiCall(
+    errors,
+    `Creating pull robot account ${fullName} with project level perms`,
+    () => robotApi.createRobot(projectRobot),
+  )) as RobotCreated
+  if (!robotPullAccount?.id) {
+    throw new Error(
```

Comment:
I am not sure if that is right wait of dealing with this error.
If I cannot create a robot account does it mean I cannot configure other parts of of Harbor ?